content/en/providers/check-in/sp/_index.md

```diff
@@ -769,6 +674,198 @@ Alternatively VO portals could implement such functionality themselves
 by using the API described at the [Master Portal sshkey endpoint
 description](https://wiki.nikhef.nl/grid/Master_Portal_sshkey_endpoint).
 
+## User attributes
+
+This section defines the attributes that can be made available to services
+connected to Check-in.
+
+### 1. EGI ID
+
+|          attribute name | EGI ID                                                                           |
+| ----------------------: | :------------------------------------------------------------------------------- |
+|         **description** | An identifier for the user, unique among all EGI accounts and never reused       |
+|   **SAML Attribute(s)** | `1.3.6.1.4.1.5923.1.1.1.13` (eduPersonUniqueId)                                  |
+|          **OIDC scope** | `openid`                                                                         |
+|       **OIDC claim(s)** | `sub `                                                                           |
+| **OIDC claim location** | - ID token <br> - Userinfo endpoint <br> - Introspection endpoint                |
+|              **origin** | Check-in assigns this attribute on user registration                             |
+|             **changes** | No                                                                               |
+|        **multiplicity** | No                                                                               |
+|        **availability** | Always                                                                           |
+|             **example** | _ef72285491ffe53c39b75bdcef46689f5d26ddfa00312365cc4fb5ce97e9ca87@egi.eu_        |
+|               **notes** | Use **EGI ID** within your application as the unique-identifier key for the user |
+|              **status** | Stable                                                                           |
+
+### 2. Display Name
+
+|          attribute name | Display Name                                      |
+| ----------------------: | :------------------------------------------------ |
+|         **description** | The user's full name, in a displayable form       |
+|   **SAML Attribute(s)** | `urn:oid:2.16.840.1.113730.3.1.241` (displayName) |
+|          **OIDC scope** | `profile`                                         |
+|       **OIDC claim(s)** | `name`                                            |
+| **OIDC claim location** | - Userinfo endpoint                               |
```

Comment:
why the `-` before Userinfo?

Reply:
It suppose 'OIDC claim location' to have a list of locations.
Since for this case, there is only one I can remove it the `-`